chill-scala/src/main/scala/com/twitter/chill/KryoBase.scala

```diff
@@ -130,7 +130,7 @@ object Instantiators {
     }
 
   // This one tries reflectasm, which is a fast way of constructing an object
-  def reflectAsm(t: Class[_]): Either[Throwable, ObjectInstantiator] = {
+  def reflectAsm(t: Class[_]): Either[Throwable, ObjectInstantiator[_]] = {
```

Comment:
same `t: Class[_ <: K]` should do it, and we can convert to `Try[ObjectInstatiator[K]]` here. There was no `Try` when we wrote this.